React/Nextjs Project Post Setup Steps.md

```diff
@@ -0,0 +1,679 @@
+## Next.js project post setup steps
```

Comment:
It goes also for other files - the title should not be in the md file because it's shown based on the filename. All other headings (e.g. Project example) should start with level 2 heading